libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/initializeDashboardHandler/index.ts

```diff
@@ -417,8 +401,11 @@ function* initializeNewDashboard(
 }
 
 export function* requestCatalog(ctx: DashboardContext) {
-    const catalog: IWorkspaceCatalog = yield call(loadCatalog, ctx, { payload: { config: {} } } as any);
-    const dateHierarchyTemplates: IDateHierarchyTemplate[] = yield call(loadDateHierarchyTemplates, ctx);
+    const [catalog, dateHierarchyTemplates]: [IWorkspaceCatalog, IDateHierarchyTemplate[]] = yield all([
+        call(loadCatalog, ctx, { payload: { config: {} } } as InitializeDashboard),
+        call(loadDateHierarchyTemplates, ctx),
```

Comment:
Just a note, loadDateHierarchyTemplates is internally a Promise.resolve() but indeed, it can be done in parallel :+1: